src/Tribe/RSVP.php

```diff
@@ -858,9 +858,16 @@ public function delete_ticket( $event_id, $ticket_id ) {
 		}
 		$product_id = get_post_meta( $ticket_id, self::ATTENDEE_PRODUCT_KEY, true );
 
-		// Decrement the sales figure
-		$sales = (int) get_post_meta( $product_id, 'total_sales', true );
-		update_post_meta( $product_id, 'total_sales', -- $sales );
+		// For attendees whose status ('going' or 'not going') for whom a stock adjustment is required?
+		$rsvp_options    = $this->tickets_view->get_rsvp_options( null, false );
+		$attendee_status = get_post_meta( $ticket_id, self::ATTENDEE_RSVP_KEY, true );
+		$adjustment      = absint( $rsvp_options[ $attendee_status ]['decrease_stock_by'] );
```

Comment:
Good point, we don't know for sure that `$attendee_status` will be set because `$rsvp_options` is filtered; will add a safety check :+1: